spec/spec_helper.rb

```diff
@@ -37,7 +37,7 @@
     ManageIQPerformance.instance_variable_set :@config, nil
 
     if defined? ManageIQPerformance::Configuration
-      ManageIQPerformance::Configuration.instance_variable_set :@config_file_location, nil
+      ManageIQPerformance::Configuration.instance_variable_set :@config_file_location, 'disabled'
```

Comment:
Maybe use [Tempfile](http://ruby-doc.org/stdlib-2.3.1/libdoc/tempfile/rdoc/Tempfile.html) here instead just incase this is somehow a legit file (#pedantic).